Add Openshift setup and test

chavafg opened the pull request:
This commit adds scripts to setup the environment for running OpenShift on top of Clear Containers, initialize the nodes and run a simple Connectivity test.

Fixes #465.

Signed-off-by: Salvador Fuentes <salvador.fuentes@intel.com>

egernst commented on integration/openshift/init.sh:
```diff
@@ -0,0 +1,53 @@
+
+echo "Disable SELinux"
+echo "There is an issue when runnig CRI-O workloads with SELinux enabled
+For more information, please take a look at:
```
nice, thanks salvador!

egernst commented on integration/openshift/openshiftrc:
```diff
@@ -0,0 +1,7 @@
+# This variables will be used for launching the cluster and
```
s/This/These

chavafg commented:
@egernst thanks for your comments. 

I added a verification on the scripts to state that currently these scripts only work on Fedora. 
Also added a README.md with that information and how to execute the scripts.

chavafg commented:
@iphutch can you take a look at the README created here? thanks

iphutch commented on integration/openshift/README.md:
```diff
@@ -0,0 +1,27 @@
+# Openshift integration with Clear Containers
```
Replace Openshift with OpenShift\*
The * is to satisfy the trademark in the first mention.

iphutch commented on integration/openshift/README.md:
```diff
@@ -0,0 +1,27 @@
+# Openshift integration with Clear Containers
+
+In this section you can find scripts to setup an Openshift enviornment
```
OpenShift should always have a capitalized S.
Could replace "In this section you can find" with "This section contains"

iphutch commented on integration/openshift/README.md:
```diff
@@ -0,0 +1,27 @@
+In this section you can find scripts to setup an Openshift enviornment
+on top of Clear Containers. Currently, these scripts only work for **Fedora**.
+
+Please execute the steps below on a Clean Environment.
```
Use lower case for "clean environment" and remove "Please" to start the sentence with an action.

iphutch commented on integration/openshift/README.md:
````diff
@@ -0,0 +1,27 @@
+$ ./setup.sh
+```
+
+Initialize the Master server and the node server:
````
Use lower case for "master"

iphutch commented on integration/openshift/README.md:
````diff
@@ -0,0 +1,27 @@
+```
+
+You can also find a basic test which can be executed in order to verify that the
+environment was build correctly. To run the test you need the `bats` framework.
````
I suggest we re-write the sentence beginning on line 20:
Run a basic verification test to check the environment was built correctly (recommended).  To run the test you need the `bats` framework.

chavafg commented:
@iphutch thanks for your comments, they are now fixed.

devimc commented on integration/openshift/hello_world.bats:
```diff
@@ -0,0 +1,52 @@
+	wait_time=20
+	sleep_time=3
+	output_file=$(mktemp)
+	echo "$output_file"
```
is this a debug message?

devimc commented:
did you plan to add a teardown script for openshift ?

devimc commented:
lgtm
